system/View/View.php

```diff
@@ -191,10 +202,12 @@ public function render(string $view, array $options = null, $saveData = null): s
 		}
 
 		ob_start();
-		include($file); // PHP will be processed
+		include($this->viewFile); // PHP will be processed
```

Comment:
Maybe all render variables must be protected to not be overwrited by custom user data.